<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="LAPTOP-A5PG7G9H" failures="2" tests="25" name="Testhigh" time="13.846" errors="0" timestamp="2022-12-28T19:39:12 IST">
  <testcase classname="testNGpackage.TestNGKeywords" name="D" time="0.033"/>
  <testcase classname="testNGpackage.TestNGAnnotataion" name="method1" time="0.031"/>
  <testcase classname="testNGpackage.Keywords2" name="logout" time="0.033"/>
  <testcase classname="testNGpackage.Keywords" name="home" time="0.032"/>
  <testcase classname="testNGpackage.TestNGKeywords" name="A" time="0.013"/>
  <testcase classname="testNGpackage.Keyworsds3" name="abc" time="0.052"/>
  <testcase classname="testNGpackage.TestNGAnnotataion" name="method2" time="0.01"/>
  <testcase classname="testNGpackage.Keywords" name="logout" time="0.005"/>
  <testcase classname="testNGpackage.TestNGAnnotataion" name="method3" time="0.004"/>
  <testcase classname="testNGpackage.Keywords2" name="login" time="0.044"/>
  <testcase classname="testNGpackage.Keywords" name="login" time="0.021"/>
  <testcase classname="testNGpackage.TestNGKeywords" name="C" time="0.034"/>
  <testcase classname="testNGpackage.Keywords" name="login" time="0.005"/>
  <testcase classname="testNGpackage.Keywords" name="login" time="0.002"/>
  <testcase classname="testNGpackage.TestNGKeywords" name="B" time="0.005"/>
  <testcase classname="testNGpackage.BeforeTestAfterAnnotation" name="method2" time="0.006"/>
  <testcase classname="testNGpackage.TestNGKeywords" name="E" time="0.003"/>
  <testcase classname="testNGpackage.Keyworsds3" name="def" time="2.029">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method testNGpackage.Keyworsds3.def() didn&amp;apos;t finish within the time-out 2000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method testNGpackage.Keyworsds3.def() didn't finish within the time-out 2000
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- def -->
  <testcase classname="testNGpackage.Keyworsds3" name="loginTest" time="0.008">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNGpackage.Keyworsds3.loginTest(Keyworsds3.java:61)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="testNGpackage.Keyworsds3" name="testA" time="0.004"/>
  <testcase classname="testNGpackage.Keyworsds3" name="login" time="0.006"/>
  <testcase classname="testNGpackage.Keyworsds3" name="C" time="0.003"/>
  <testcase classname="testNGpackage.Keyworsds3" name="B" time="0.003"/>
  <testcase classname="testNGpackage.Keyworsds3" name="logout" time="0.002"/>
  <testcase classname="testNGpackage.automation1" name="method1" time="13.634"/>
  <testcase classname="testNGpackage.Keywords2" name="home" time="0.0">
    <ignored/>
  </testcase> <!-- home -->
  <testcase classname="testNGpackage.Keyworsds3" name="testB" time="0.0">
    <ignored/>
  </testcase> <!-- testB -->
</testsuite> <!-- Testhigh -->
